sonar-python-plugin/src/test/java/org/sonar/plugins/python/IPynbProfileTest.java

```diff
@@ -38,5 +38,8 @@ public void profile() {
     assertThat(profile.rules()).extracting("repoKey").containsOnly("ipython");
     assertThat(profile.rules()).hasSizeGreaterThan(25);
     assertThat(profile.rules()).extracting(BuiltInQualityProfilesDefinition.BuiltInActiveRule::ruleKey).contains("S100");
+    assertThat(profile.rules()).extracting(BuiltInQualityProfilesDefinition.BuiltInActiveRule::ruleKey).doesNotContain("S905");
```

Comment:
what about having a test similar to `IPynbSensorTest` where we assert issues for those rules are not reported?

Reply:
I'm not sure I understand. In our sensor tests we generally manually activate the rules we want to enable. If the rule is enabled, I would expect it to raise an issue. Here, we simply test that the rule is left out of the default quality profile.
Basically what I'm testing is that the default quality profile for IPython is the same as the one for Python, except for those rules we filter out. My assumption is that, for now, we will not deliver new rules in IPython that would be excluded from Python (making IPython profile a strict subset of the Python one).

Reply:
I was thinking having a test where we activate all default quality profile rules and assert no issue is raised for those 3 rules. I don't think it's strictly necessary however it can makes things more explicit, wdyt?

Reply:
I am under the impression this would not be so easy to do outside of an integration test. In our regular sensor tests, we build the check factory manually, with the rules activated manually as well.